xml/System.Drawing.Imaging.Effects/Effect.xml

```diff
@@ -65,8 +65,8 @@
         <Parameter Name="disposing" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="disposing">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="disposing"><see langword="true"/> if managed resources should be disposed; otherwise, <see langword="false"/>.</param>
+        <summary>Releases the resources that are used by the object.</summary>
```

Comment:
What do you think of this alternative description? It's like the one we use for other similar `Dispose(bool)` methods:

```suggestion
        <param name="disposing">
          <see langword="true" /> to release both managed and unmanaged resources; false to release only unmanaged resources.</param>
        <summary>Releases the unmanaged resources used by the <see cref="T:System.Drawing.Imaging.Effects.Effect" /> and optionally releases the managed resources.</summary>
```